lib/themes/dosomething/paraneue_dosomething/js/takeover/Takeover.js

```diff
@@ -0,0 +1,245 @@
+import Validation from 'dosomething-validation';
+import setting from '../utilities/Setting';
+
+// @IDEA - Let this just take care of screen interactions, pull in custom stuff via classes
+import takeoverTpl from 'takeover/templates/takeover.tpl.html';
+
+const $ = require('jquery');
+const template = require('lodash/string/template');
+
+const Takeover = {
+  baseUrl: $(location).attr("origin"),
```

Comment:
Is there an advantage to wrapping `location` in a jQuery element?

If not, I think `window.location.origin` might be more clear.